src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php

```diff
@@ -126,10 +126,20 @@ protected function getDefaultFormatter(): FormatterInterface
     }
 
     /**
-     * {@inheritdoc}
+     * @param \Monolog\LogRecord|array $record
      */
-    protected function write(array $record): void
+    protected function write($record): void
     {
+        if (version_compare(PHP_VERSION, '8', '>=')) {
+            if($record instanceof \Monolog\LogRecord){
```

Comment:
plese use `use Monolog\LogRecord`; instead of FQDN

Reply:
handling different version  of php lead to this if statement  did use FQDN to avoid importing unrecognizable dependencies for older version.

Reply:
in PHP it's totally fine to `use` a class/interface that does not exist.

You can even use the `TheClass::class` in your code.

see https://3v4l.org/QclWd

Reply:
fixed